package.json

```diff
@@ -32,16 +32,16 @@
   "private": true,
   "dependencies": {
     "@balena/happy-eyeballs": "0.0.6",
-    "dbus": "^1.0.7",
+    "@balena/systemd": "^0.4.0",
```

Comment:
Can this update be merged prior to the nodejs bump? That way each individual PR/change can be smaller and tested in isolation in case any issues do arise

Reply:
Unfortunately no, we need at least rust v1.64 to be able to compile the module (because of dependencies), but alpine 3.17, which comes with that rust version, also comes with nodejs 18

Reply:
Is the rust part not precompiled as part of the module's CI?

Reply:
No, `@balena/systemd` is not a WASM module, is just some bindings to node NAPI. It has to be compiled when doing `npm install` so the build of the module matches the architecture of the host, similar to what we do with `sqlite3`

Reply:
I though you can still have prebuilds for a bunch of architectures and it'll only fallback to a local build if it doesn't find a prebuilt one that matches?

Reply:
You might be able to do it, I haven’t really looked into that option. I’ll investigate a bit